documentation/docs/faq/market-making.md

```diff
@@ -7,7 +7,59 @@ Market makers play an important role in providing liquidity to financial markets
+
+![Figure 2: A clam market with regular trading activity](/assets/img/pure-mm-calm.png)
+
+In the figure above, the period between 25 Feb and 12 Mar would be an example of the ideal case. The price of the asset stayed within a relatively small range, and there was sufficient trading activity for a market maker's offers to be taken regularly.
+
+The only thing a market maker needs to worry about in this scenario is he must make sure the trading spread he sets is larger than the trading fees given to the exchange.
+
+### Low trading activity
+
+Markets with low trading activity higher risk for pure market making strategies. Here's an example:
+
+![Figure 3: A market with low trading activity](/assets/img/pure-mm-low-volume.png)
+
+In any market with low trading activity, there's a risk where the market maker may need to hold onto inventory for a long time without a chance to trade it back. During that time, the prices of the traded assets may rise or drop dramatically despite seeing no or little trading activity on the exchange. This exposes the market maker to inventory risk, even after mitigating some of this risk by using wider bid spreads.
+
+Other strategies may be more suitable from a risk perspective in this type of market, e.g. [cross-exchange market making](/strategies/cross-exchange-market-making).
+
+### Volatile or trending markets
+
+Another common risk that market makers need to be aware of is trending markets. Here's one example:
+
+![Figure 4: A trending market](/assets/img/pure-mm-trending.png)
+
+If a pure market maker set his spreads naively in such a market, e.g. equidistant bid/ask spread, there's a risk of the market maker's bid consistently being filled as prices trend down, while at the same time the market continues to move away from the market maker's ask, decreasing the probability of sells.  This would result in an accumulation of inventory at exactly the time where this would reduce inventory inventory value, which is "wrong-way" risk.
+
+However, it is still possible to improve the probability of generating profits in this kind of market by skewing bid asks, i.e. setting a wider bid spread (e.g. -4%) than ask spread (e.g. +0.5%).  In this way, the market maker is trying to catch price spikes in the direction of the trend and buy additional inventory only in the event of a larger moves, but sell more quickly when there is an opportunity so as to minimize the duration the inventory is held.  This approach also has a mean reversion bias, i.e. buy only when there is a larger move downwards, in the hopes of stabilization or recovery after such a large move.
+
+Market making in volatile or trending markets is more advanced and risky for new traders. It's recommended that a trader looking to market make in this kind of environment to get mentally familiar with it (e.g. via paper trading) before committing meaningful capital to the strategy.
+
+## What risks does a market maker face?
```

Comment:
I would say: "Besides market risk, what other risks does a market maker face?"

If users search for this, they may not read the previously mentioned risks.